components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -744,10 +744,11 @@ bool PublisherInfoDatabase::GetExcludedList(
   }
 
   // We will use every attribute from publisher_info
-  std::string query = "SELECT * FROM publisher_info WHERE excluded = 1";
+  std::string query = "SELECT publisher_id, verified, excluded, "
```

Comment:
excluded is not used, so we don't need to select it

Reply:
updated